pymc3/distributions/multivariate.py

```diff
@@ -112,17 +113,13 @@ def quaddist_parse(value, mu, cov, mat_type="cov"):
         onedim = False
 
     delta = value - mu
-
-    if mat_type == "cov":
-        # Use this when Theano#5908 is released.
-        # return MvNormalLogp()(self.cov, delta)
-        chol_cov = cholesky(cov)
+    # Use this when Theano#5908 is released.
+    # return MvNormalLogp()(self.cov, delta)
+    chol_cov = cholesky(cov)
+    if mat_type == "cov" or mat_type != "tau":
```

Comment:
```suggestion
    if mat_type != "tau":
```
Just a minor suggestion, feel free to merge if you want